Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
 # cilium-envoy from github.com/cilium/proxy
 #
-FROM quay.io/cilium/cilium-envoy:84ee839e1d78ef858a39e390288ad417d35bb1d4 as cilium-envoy
+FROM quay.io/cilium/cilium-envoy:9e1a3317863a5dd0e393fd168ce85cc2a3c709e4 as cilium-envoy
```

Comment:
These two SHAs may sometimes be out of sync. The Godep dependency is for the API (protobufs), and it may not need to be updated everytime something is fixed in the proxy. Dockerfile refers to the cilium-envoy build and needs to change when bugs are fixed or new features are added to the proxy.